### PR TITLE
Header nav

### PR DESCRIPTION
**Pull request by VasylDolishnii**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,93 +1,59 @@
...
\ No newline at end of file
+      </section>
+      <include src="./partials/hero.html"></include>
+      <!-- mobile-menu END -->
+    </header>
+    <main></main>
+    <footer></footer>
+
+    <!-- mobile-menu start /pod footer/-->
+    <include src="./partials/humburger-after-click.html"></include>
```
Comment (matilis) [inline src/index.html:47]: delete this include. matilis added new

File: src/index.html [hunk 2]
```diff
@@ -1,93 +1,59 @@
...
+
+    <script
```
Comment (matilis) [inline src/index.html:49]: delete this script, matilis added new include

File: src/partials/mobile-menu.html [hunk 3]
```diff
@@ -1,11 +1,19 @@
 <!-- mobile-menu start-->
 <div class="hamburger">
-    <button class="menu-toggle js-open-menu" aria-expanded="false" aria-controls="mobile-menu">
-        <svg width="20" height="20" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
-            <path fill-rule="evenodd"
-                d="M3 5a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 10a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 15a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1z"
-                clip-rule="evenodd"></path>
-        </svg>
-    </button>
+  <button class="menu-toggle js-open-menu" aria-expanded="false" aria-controls="mobile-menu">
```
Comment (matilis) [inline src/partials/mobile-menu.html:3]: delete this, matilis added this section

File: src/sass/partials/_mobile-menu.scss [hunk 4]
```diff
@@ -5,7 +5,7 @@
 }
 
 .current {
```
Comment (matilis) [inline src/sass/partials/_mobile-menu.scss:7]: !important is not necessary. Matilis did this, You can deleted

File: src/sass/partials/_mobile-menu.scss [hunk 5]
```diff
@@ -5,7 +5,7 @@
...
-  color: $IIIColor;
+  color: $IIIColor !important;
 }
 
 .menu-container {
```
Comment (matilis) [inline src/sass/partials/_mobile-menu.scss:11]: matilis fixed this all section, you can deleted all mobile-menu section

**Review by matilis — COMMENTED** [review]
we need to meet and discuss the header

**Review by WojciechDutkiewicz — APPROVED** [review]

**Comment by KJachimstal** [comment]
Outdated

**Pull request closed without merging**